src/compiler/commandLineParser.ts

```diff
@@ -502,6 +502,26 @@ export const commonOptionsWithBuild: CommandLineOption[] = [
         description: Diagnostics.Include_sourcemap_files_inside_the_emitted_JavaScript,
         defaultValueDescription: false,
     },
+    {
+        name: "noCheck",
+        type: "boolean",
+        showInSimplifiedHelpView: false,
+        category: Diagnostics.Compiler_Diagnostics,
+        description: Diagnostics.Disable_full_type_checking_only_critical_parse_and_emit_errors_will_be_reported,
+        transpileOptionValue: true,
+        defaultValueDescription: false,
+        affectsSemanticDiagnostics: true,
+        affectsBuildInfo: true,
+    },

```

Comment:
Is there an option here to prevent it from being set in a tsconfig, or are we okay with it being there?

Reply:
With this change it doesnt matter whether its in tsconfig or commandline

Reply:
Sure, I'm just trying to understand who'd intentionally set that in a tsconfig. Probably some use for it...

Reply:
Its like `noEmit` right only opposite :) who want to use tsc for transpilation i guess